ts/components/dropdowns/dropdown_products.tsx

```diff
@@ -16,6 +16,11 @@ const navData = [
         description: 'Embed simple token purchasing in any app or website',
         url: WebsitePaths.Instant,
     },
+    {
+        title: '0x Mesh',
+        description: 'Access token markets, permissionlessly',
```

Comment:
for some reason, it renders the title with a capital "P". Any way we can change that?